tests/Framework/MockResponseClient.php

```diff
@@ -26,9 +26,12 @@ public function __construct(array $config = array())
         $path = PIWIK_INCLUDE_PATH . CapturingGoogleClient::PATH_TO_CAPTURED_DATA_FILE;
         foreach (new \SplFileObject($path) as $line) {
             $decoded = json_decode($line, $isAssoc = true);
+            if (empty($decoded)) {
+                continue;
+            }
 
-            $key = md5(json_encode($decoded[0]));
-            $value = unserialize($decoded[1]);
+            $key = isset($decoded[0]) ? md5(json_encode($decoded[0])) : '';
```

Comment:
@snake14 is this failing somewhere ?

Reply:
I mean do we need this change ?

Reply:
I can try reverting it. It might be left over from when I was trying to work around the library upgrade.

Reply:
Yep. Good call. Looks like it's working fine without those changes.